pkgs/by-name/ka/kanidm/package.nix

```diff
@@ -81,12 +90,15 @@ rustPlatform.buildRustPackage rec {
     mv $out/lib/libpam_kanidm.so $out/lib/pam_kanidm.so
   '';
 
+  doCheck = false;
```

Comment:
Was this for local building or is this on purpose?

Reply:
Oh. Thanks for noticing, that was by mistake and shouldn't have been included! Sorry